### PR TITLE
[SPIR-V] Handle vk::ext_capability and vk::ext_extension on entry point

### PR DESCRIPTION
**Pull request by cassiebeckley**
`ext_capability` and `ext_extension` were previously only handled when translating a CallExpr. Since the entry point is never called, they were ignored when attached to it.

### CODE REVIEW

**Review by sudonatalie — APPROVED** [review]
Looks good to me!

File: tools/clang/lib/SPIRV/SpirvEmitter.cpp [hunk 1]
```diff
@@ -13191,6 +13196,8 @@ bool SpirvEmitter::emitEntryFunctionWrapper(const FunctionDecl *decl,
   auto *entryLabel = spvBuilder.createBasicBlock();
   spvBuilder.setInsertPoint(entryLabel);
 
+  processInlineSpirvAttributes(decl);
+
```
Comment (s-perron) [inline tools/clang/lib/SPIRV/SpirvEmitter.cpp:13200]: Why did you move the call to this function? I'm fine with it here, but if it needs to be called after something else happens, then we should document it with a comment.

**Review by s-perron — APPROVED** [review]

File: tools/clang/lib/SPIRV/SpirvEmitter.cpp (hunk 1 above)
Comment (cassiebeckley) [reply tools/clang/lib/SPIRV/SpirvEmitter.cpp:13204]: Sure, I added a comment:

> Handle vk::execution_mode, vk::ext_extension and vk::ext_capability attributes. Uses pseudo-instructions for extensions and capabilities, which are added to the beginning of the entry basic block, so must be called after the basic block is created and insert point is set.

**Review by s-perron — APPROVED** [review]
